templates/search/blocks/search_results_tag_frequency.html

```diff
@@ -22,7 +22,7 @@
                 <a href="{% tag_type_url tag_type=tag_type.PERSON|upper %}#tagTop" class="ohos-tag ohos-tag--person">
 
                     <span class="ohos-tag__inner">
-                        Person
+                        <label for="Category: Person">Person</label>
```

Comment:
Label is usually associated with an input field. (same browser error as label for Category Location

Reply:
now using aria-label on <a href> tags